pkg/fd/fd.go

```diff
@@ -0,0 +1,109 @@
+//go:build !windows
+// +build !windows
+
+// This file has been adapted from https://github.com/ftrvxmtrx/fd/blob/master/fd.go
```

Comment:
```suggestion
// This file has been adapted from https://github.com/ftrvxmtrx/fd/blob/c6d800382fff6dc1412f34269f71b7f83bd059ad/fd.go

```

Also please follow https://github.com/ftrvxmtrx/fd/blob/c6d800382fff6dc1412f34269f71b7f83bd059ad/LICENSE.MIT

Reply:
Maybe you can just fork https://github.com/ftrvxmtrx/fd into your own repo, and just import it here

Reply:
Will do this.